Patch to exclude firmware installation with library.
==========================================================================
--- a/linux_lib/libcrystalhd/Makefile
+++ b/linux_lib/libcrystalhd/Makefile
@@ -53,14 +53,11 @@
 
 install:
 	mkdir -p $(DESTDIR)$(LIBDIR)
-	mkdir -p $(DESTDIR)/lib/firmware
 	mkdir -p $(DESTDIR)/usr/include/libcrystalhd
 	cp libcrystalhd_if.h $(DESTDIR)/usr/include/libcrystalhd/
 	cp $(ROOTDIR)/include/bc_dts_defs.h $(DESTDIR)/usr/include/libcrystalhd/
 	cp $(ROOTDIR)/include/bc_dts_types.h $(DESTDIR)/usr/include/libcrystalhd/
 	cp $(ROOTDIR)/include/libcrystalhd_version.h $(DESTDIR)/usr/include/libcrystalhd/
-	cp $(ROOTDIR)/firmware/fwbin/70012/bcm70012fw.bin $(DESTDIR)/lib/firmware/
-	cp $(ROOTDIR)/firmware/fwbin/70015/bcm70015fw.bin $(DESTDIR)/lib/firmware/
 	install -m 755 $(BCLIB) $(DESTDIR)$(LIBDIR)
 	(cd $(DESTDIR)$(LIBDIR); ln -sf $(BCLIB) $(BCLIB_NAME))
 	(cd $(DESTDIR)$(LIBDIR); ln -sf $(BCLIB) $(BCLIB_SL))
